reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -122,7 +132,8 @@ class FreemarkerTemplateProcessor(
             "LicenseView" to LicenseView,
             "helper" to TemplateHelper(input),
             "projectsAsPackages" to projectsAsPackages,
-            "vulnerabilityReference" to VulnerabilityReference
+            "vulnerabilityReference" to VulnerabilityReference,
+            "tabularScanRecord" to tabularScanRecord
```

Comment:
Instead of adding `tabularScanRecord` to dataModel, I propose we move it to `helper` so we have a cleaner API signature.

Reply:
I have implemented this as requested (I think)